packages/augur-sdk/src/state/getter/Markets.ts

```diff
@@ -414,106 +442,153 @@ export class Markets {
+      filteredKeyedMarketCreatedLogs = Object.values(
         keyedMarketCreatedLogs
       ).reduce((previousValue: any, currentValue: any) => {
-        if (keyedFullTextMarketIds[currentValue[0]]) {
-          previousValue[currentValue[0]] = currentValue[1];
+        if (keyedFullTextResults[currentValue.market]) {
+          previousValue[currentValue.market] = currentValue;
         }
         return previousValue;
       }, []);
     }
 
-    await Promise.all(
-      Object.entries(filteredKeyedMarketCreatedLogs).map(
-        async (marketCreatedLogInfo: any) => {
-          let includeMarket = true;
-
-          if (params.disputeWindow) {
-            const market = await augur.contracts.marketFromAddress(
-              marketCreatedLogInfo[0]
-            );
-            const disputeWindowAddress = await market.getDisputeWindow_();
-            if (params.disputeWindow !== disputeWindowAddress) {
-              includeMarket = false;
-            }
-          }
+    let filteredMarketsInfo: any[] = [];
+    for (const marketCreatedLogInfo of Object.values(filteredKeyedMarketCreatedLogs)) {
+      let includeMarket = true;
 
-          // TODO: when currentOrders event table exists just check that
-          if (params.hasOrders) {
-            const orderCreatedLogs = await db.findOrderCreatedLogs({
-              selector: { market: marketCreatedLogInfo[0] },
-            });
-            const orderCanceledLogs = await db.findOrderCanceledLogs({
-              selector: { market: marketCreatedLogInfo[0] },
-            });
-            const orderFilledLogs = await db.findOrderFilledLogs({
-              selector: { market: marketCreatedLogInfo[0], amount: '0x00' },
-            });
-            if (
-              orderCreatedLogs.length - orderCanceledLogs.length ===
-              orderFilledLogs.length
-            ) {
-              includeMarket = false;
-            }
-          }
+      if (params.disputeWindow) {
+        const market = await augur.contracts.marketFromAddress(
+          marketCreatedLogInfo['market']
+        );
+        const disputeWindowAddress = await market.getDisputeWindow_();
+        if (params.disputeWindow !== disputeWindowAddress) {
+          includeMarket = false;
+        }
+      }
 
-          if (params.reportingState) {
-            const reportingStates = Array.isArray(params.reportingState)
-              ? params.reportingState
-              : [params.reportingState];
-            const marketFinalizedLogs = await db.findMarketFinalizedLogs({
-              selector: { market: marketCreatedLogInfo[0] },
-            });
-            const reportingState = await getMarketReportingState(
-              db,
-              marketCreatedLogInfo[1],
-              marketFinalizedLogs
-            );
-            if (!reportingStates.includes(reportingState)) {
-              includeMarket = false;
-            }
+      if (params.includeMarketsWithoutOrders === false) {
```

Comment:
This got removed by design. UI won't be using this filter. The liquidity filter replace `no orders` filter.

Reply:
Ok, will remove this then.